src/sql/plot.py

```diff
@@ -297,8 +297,7 @@ def _min_max(con, table, column, with_=None, use_backticks=False):
 
     template = Template(template_)
     query = template.render(table=table, column=column)
-
-    min_, max_ = con.execute(query, with_).fetchone()
+    min_, max_ = con.execute(query, with_=with_, read="mysql").fetchone()
```

Comment:
why is mysql hardcoded?

Reply:
Removed the read parameter, we don't need that anymore